CMakeLists.txt

```diff
@@ -91,9 +91,7 @@ add_library(glad ${GLAD_SOURCES})
 add_dependencies(glad glad-generate-files)
 target_include_directories(glad
     PUBLIC
-        $<BUILD_INTERFACE:${GLAD_INCLUDE_DIRS}>
-    INTERFACE
-        $<INSTALL_INTERFACE:include/glad>)
```

Comment:
Why can this be removed?

Reply:
This is one option, but not required.  Normally if you have a common header folder you can use the PUBLIC_HEADER property, and then include PUBLIC_HEADER in the INSTALL declaration.  Given the headers go into two different folders this isn't an option.  The "install(FILES" explicitly selects the files.

Reply:
I'm pretty sure it must not be removed, it's an information for the generated config files to set the include property on an installed package. But it was wrong, it should have been ` $<INSTALL_INTERFACE:include>`